include/lsst/afw/geom/Transform.h

```diff
@@ -220,7 +220,6 @@ class Transform {
     */
     explicit Transform(std::shared_ptr<ast::FrameSet> &&frameSet);
 
-private:
```

Comment:
I want efficient access without allowing users to mess with the internals. Thus I can't expose the pointers with public accessors. I could add protected accessors but don't see enough advantage to that over just making the instance variables protected.